BackofficeBundle/Resources/public/coffee/OrchestraView.coffee

```diff
@@ -94,4 +94,16 @@ OrchestraView = Backbone.View.extend(
       self_duplicate: element.get('links')._self_duplicate
       path: path.duplicate
     ) if element.get('links')._self_duplicate
+
+  addConcurrency: ->
+    if typeof @currentRankView == 'undefined'
+      @oldRender = @render
+      @currentRankView = 0
+    else
+      @currentRankView = @currentRankView + 1
+    @render = () ->
```

Comment:
why not add an abstract view than the different channel extend it. In this abstract view you could add this this behavior on the render.